src/ktopt_planning_context.cpp

```diff
@@ -33,35 +35,36 @@ void KTOptPlanningContext::solve(planning_interface::MotionPlanDetailedResponse&
 
 void KTOptPlanningContext::solve(planning_interface::MotionPlanResponse& res)
 {
-  RCLCPP_INFO(getLogger(),
-              "Setting up and solving optimization problem ..."
-              );
+  RCLCPP_INFO(getLogger(), "Setting up and solving optimization problem ...");
   // preliminary house keeping
   const auto time_start = std::chrono::steady_clock::now();
   res.planner_id = std::string("ktopt");
   res.error_code.val = moveit_msgs::msg::MoveItErrorCodes::SUCCESS;
 
+  // dome drake related scope initialisations
+  const auto& plant_ = 
```

Comment:
Only class member variables should have the `_` suffix. Do you mind removing that in all places in this file?